src/main/java/com/example/demo/service/impl/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package com.example.demo.service.impl;
+
+import com.example.demo.exceptions.ReadFileException;
+import com.example.demo.service.FileReaderService;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.springframework.stereotype.Service;
+
+@Service
+public class FileReaderServiceImpl implements FileReaderService {
+    @Override
+    public List<String> read(String path) throws ReadFileException {
```

Comment:
do we need to indicate that method throws RuntimeException?

Reply:
done